torch_uncertainty/routines/regression.py

```diff
@@ -88,7 +117,13 @@ def training_step(
         inputs, targets = batch
         logits = self.forward(inputs)
 
-        if self.dist_estimation:
+        if self.dist_estimation == 4:
+            means, v, alpha, beta = logits.split(1, dim=-1)
```

Comment:
In the longer term, we have to think of some way to specify the distribution that we want to use, although it is true that distributions with four parameters per dim are likely to remain NIGs.

Reply:
I think it would be better to temporarily assume that the `dist_estimation` argument is equal to the `num_outputs` argument and set them to be the same. What are your thoughts?

Reply:
I had the same kind of problem with another project of mine, I would suggest creating a function like this:

```py
def get_distribution(dist_name: str, dist_params: Tensor) -> Distribution:
    ...
```
I think it could be interesting to use the `Distribution` class from Pytorch, but it might be too much. A dictionary with the parameters should do it nicely too.

Reply:
Sure, I'll begin working on it. Do I need to open a separate issue for this, or can I work on it within this PR?